services/dados-publicos-senado/servidores/index.js

```diff
@@ -0,0 +1,133 @@
+import NotFoundError from '@/errors/NotFoundError';
+import axios from 'axios';
+
+const API_URL = 'https://adm.senado.gov.br/adm-dadosabertos/api/v1/servidores'; 
+
+export const getServidores = async (tipoVinculoEquals, situacaoEquals, lotacaoEquals, cargoEquals) => {
+  
+  var params = "";
+
+  if(tipoVinculoEquals != null)
+    params = `tipoVinculoEquals=${tipoVinculoEquals}`;
+
+  if(situacaoEquals != null)
+    params += (params == "") ? `tipoVinculoEquals=${situacaoEquals}` : `&tipoVinculoEquals=${situacaoEquals}`;
+
+  if(lotacaoEquals != null)
+    params += (params == "") ? `tipoVinculoEquals=${lotacaoEquals}` : `&tipoVinculoEquals=${lotacaoEquals}`;
+
+  if(cargoEquals != null)
+    params += (params == "") ? `tipoVinculoEquals=${lotacaoEquals}` : `&tipoVinculoEquals=${lotacaoEquals}`
+
+  if(params != "")
+    params = "?" + params;
+
+  return axios.get(API_URL +'/servidores' + params);
```

Comment:
Sugiro fazer algo mais ou menos assim:
```suggestion
  return axios.get(API_URL +'/servidores', {
    params: {
      tipoVinculoEquals,
      situacaoEquals,
      lotacaoEquals,
      cargoEquals,
    }
  });
```

https://masteringjs.io/tutorials/axios/get-query-params